### PR TITLE
Bug fixes non breaking

### PR DESCRIPTION
**Pull request by innominata**
Fixed vein detection on small planets. Adjusted graphics of miner radius to follow curvature of ground.
Avoided game crash when trying to create planets without a corresponding orbit, by creating new orbit and printing error to console.
Fixed error when placing belts from ghost buildings
Fixed Tidally Locked planets being able to have retrograde orbits. (May potentially move planets in space, I haven't found this to be the case) Tidally locked planets now don't rotate.

### CODE REVIEW

File: DSP_Plugins.GalacticScale/DSP_Plugins.GalacticScale.csproj [hunk 1]
```diff
@@ -38,7 +38,7 @@
       <HintPath>..\Libs\0Harmony.dll</HintPath>
     </Reference>
     <Reference Include="Assembly-CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null">
-      <HintPath>G:\SteamLibrary\steamapps\common\Dyson Sphere Program\DSPGAME_Data\Managed\Assembly-CSharp.dll</HintPath>
+      <HintPath>..\..\..\Program Files (x86)\Steam\steamapps\common\Dyson Sphere Program\DSPGAME_Data\Managed\Assembly-CSharp.dll</HintPath>
```
Comment (Touhma) [inline DSP_Plugins.GalacticScale/DSP_Plugins.GalacticScale.csproj:41]: do not change this one

File: DSP_Plugins.GalacticScale/Scripts/PatchPlanetSize/PatchNearColliderLogic.cs [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    {
+        public static int offset = -20;
+        public static List<GameObject> markers;
+
+        [HarmonyPostfix]
+        [HarmonyPatch("Update")]
+        public static void Update()
+        {
+            int o = offset;
+            if ((Input.GetKeyUp(KeyCode.Equals) || Input.GetKeyUp(KeyCode.KeypadPlus)) && VFInput.alt)
```
Comment (Touhma) [inline DSP_Plugins.GalacticScale/Scripts/PatchPlanetSize/PatchNearColliderLogic.cs:20]: what is it for ?

File: DSP_Plugins.GalacticScale/Scripts/PatchPlanetSize/PatchNearColliderLogic.cs [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+            if ((Input.GetKeyUp(KeyCode.Minus) || Input.GetKeyUp(KeyCode.KeypadMinus)) && VFInput.alt)
+                --offset;
+            if (o != offset) Patch.Debug("Mining AreaRadius Offset Changed to " + offset, BepInEx.Logging.LogLevel.Message, true);
+
+        }
+        [HarmonyPostfix]
+        [HarmonyPatch("MarkActivePos")]
+        public static void MarkActivePos(Vector3 pos, ref NearColliderLogic __instance, ref int[] ___activeColHashes, ref int ___activeColHashCount)
+        {  
+            if (VFInput.alt && Input.GetKeyUp(KeyCode.Delete))
```
Comment (Touhma) [inline DSP_Plugins.GalacticScale/Scripts/PatchPlanetSize/PatchNearColliderLogic.cs:31]: why input needed?

**Review by Touhma — CHANGES_REQUESTED** [review]

File: DSP_Plugins.GalacticScale/Scripts/PatchPlanetSize/PatchNearColliderLogic.cs (hunk 3 above)
Comment (innominata) [reply DSP_Plugins.GalacticScale/Scripts/PatchPlanetSize/PatchNearColliderLogic.cs:31]: Debugging mostly, I could comment those out but I would prefer not to delete them entirely- for next time they change the file and we need to tweak things.

File: DSP_Plugins.GalacticScale/Scripts/PatchPlanetSize/PatchNearColliderLogic.cs (hunk 2 above)
Comment (innominata) [reply DSP_Plugins.GalacticScale/Scripts/PatchPlanetSize/PatchNearColliderLogic.cs:20]: Increases and Decreases the vein search area node size, allows you to use alternative miners

File: DSP_Plugins.GalacticScale/DSP_Plugins.GalacticScale.csproj (hunk 1 above)
Comment (innominata) [reply DSP_Plugins.GalacticScale/DSP_Plugins.GalacticScale.csproj:41]: Yeah I can't seem to ignore that file. It's in my gitignore, but keeps uploading

File: DSP_Plugins.GalacticScale/Scripts/PatchPlanetSize/PatchNearColliderLogic.cs (hunk 3 above)
Comment (Shad0wlife) [reply DSP_Plugins.GalacticScale/Scripts/PatchPlanetSize/PatchNearColliderLogic.cs:31]: You could put preprocessor conditions around it. (#if DEBUG and #endif)

That way the code would only be compiled into febug builds.

File: DSP_Plugins.GalacticScale/DSP_Plugins.GalacticScale.csproj (hunk 1 above)
Comment (Shad0wlife) [reply DSP_Plugins.GalacticScale/DSP_Plugins.GalacticScale.csproj:41]: I can show you how to circumcent this so you dont have to manually change the path everytime. I think I even talked to glitch about it on discord.

**Pull request closed without merging**